docs/documentation/release_notes/topics/26_1_0.adoc

```diff
@@ -108,6 +108,22 @@ The key providers that allow to import externally generated keys (`rsa` and `jav
 
 The default `generated` key providers generate a certificate valid for 10 years (the types that have or can have an associated certificate). Because of the long validity and the recommendation to rotate keys frequently, the generated providers do not perform this check.
 
+= OpenTelemetry Tracing supported
+
+In the previous release, the OpenTelemetry Tracing feature was preview and is *fully supported* now.
+It means the `opentelemetry` feature is enabled by default.
+
+There were made multiple improvements to the tracing capabilities in {project_name} such as:
+
+* Introduced private (for now) *Tracing SPI* for instrumenting {project_name} codebase
```

Comment:
After merging https://github.com/keycloak/keycloak/pull/35703, there will be also a paragraph about configuration via Keycloak CR for the Operator.